openlibrary/templates/lib/nav_foot.html

```diff
@@ -6,52 +6,52 @@
       <div>
         <h2>Open Library</h2>
         <ul>
-          <li><a href="/about/vision">$_('Vision')</a></li>
-          <li><a href="/volunteer">$_('Volunteer')</a></li>
-          <li><a href="/partner-with-us">$_('Partner With Us')</a></li>
-          <li><a href="https://archive.org/about/jobs.php" title="$_('Jobs')">$_('Careers')</a></li>
-          <li><a href="https://blog.openlibrary.org/">$_('Blog')</a></li>
-          <li><a href="https://archive.org/about/terms.php">$_('Terms of Service')</a></li>
-          <li><a href="https://archive.org/donate/?platform=ol&origin=olwww-TopNavDonateButton">$_('Donate')</a></li>
+          <li><a href="/about/vision">$_("Vision")</a></li>
+          <li><a href="/volunteer">$_("Volunteer")</a></li>
+          <li><a href="/partner-with-us">$_("Partner With Us")</a></li>
+          <li><a href="https://archive.org/about/jobs.php" title="$_('Jobs')">$_("Careers")</a></li>
+          <li><a href="https://blog.openlibrary.org/">$_("Blog")</a></li>
+          <li><a href="https://archive.org/about/terms.php">$_("Terms of Service")</a></li>
+          <li><a href="https://archive.org/donate/?platform=ol&origin=olwww-TopNavDonateButton">$_("Donate")</a></li>
         </ul>
       </div>
       <div>
         <h2>$:_('Discover')</h2>
         <ul>
-          <li><a href="/" title="$_('Go home')">$_('Home')</a></li>
-          <li><a href="/search" title="$_('Explore Books')">$_('Books')</a></li>
-          <li><a href="/search/authors" title="$_('Explore authors')">$_('Authors')</a></li>
-          <li><a href="/subjects" title="$_('Explore subjects')">$_('Subjects')</a></li>
-          <li><a href="/collections" title="$_('Explore collections')">$_('Collections')</a></li>
-          <li><a href="/advancedsearch">$_('Advanced Search')</a></li>
-          <li><a href="#top" title="$_('Navigate to top of this page')">$_('Return to Top')</a></li>
+          <li><a href="/" title="$_('Go home')">$_("Home")</a></li>
+          <li><a href="/search" title="$_('Explore Books')">$_("Books")</a></li>
+          <li><a href="/search/authors" title="$_('Explore authors')">$_("Authors")</a></li>
+          <li><a href="/subjects" title="$_('Explore subjects')">$_("Subjects")</a></li>
+          <li><a href="/collections" title="$_('Explore collections')">$_("Collections")</a></li>
+          <li><a href="/advancedsearch" title="$_('Advanced search')">$_("Advanced Search")</a></li>
```

Comment:
Keeping these consistent means translators only have to translate them once, which is nice!
```suggestion
          <li><a href="/advancedsearch" title="$_('Advanced Search')">$_("Advanced Search")</a></li>
```